satpy/sentinel1-false-color.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# script by Martin Raspaud\n",
+    "# modified by Mikhail Itkin\n",
+    "\n",
+    "from datetime import datetime\n",
+    "\n",
+    "from satpy import Scene\n",
+    "from satpy import DatasetID\n",
+    "from satpy.utils import debug_on\n",
+    "from trollimage.image import Image\n",
+    "\n",
+    "\n",
+    "def logenh(data):\n",
+    "    res = (np.log10(data / 5000.) - np.log10(0.0223)) /  (1.0 - np.log10(0.0223))\n",
+    "    res[res < 0] = 0\n",
+    "    return res\n",
+    "\n",
+    "\n",
+    "if __name__ == '__main__':\n",
+    "\n",
+    "    debug_on()\n",
+    "    scn = Scene(\n",
+    "        sensor='sar-c',\n",
+    "        start_time=datetime(2017, 9, 19, 0, 0),\n",
+    "        base_dir=\".\"\n",
+    "    )\n",
+    "\n",
+    "    composite = 'hv'\n",
+    "\n",
+    "    scn.load([DatasetID(name='measurement', polarization='hh'),\n",
+    "              DatasetID(name='measurement', polarization='hv')])\n",
+    "\n",
+    "    mx = 2000.\n",
+    "\n",
+    "    i = scn.datasets.values()[0]\n",
+    "    m = scn.datasets.values()[1]\n",
+    "\n",
+    "    rs = mx - (mx - m) * (mx - i) / mx\n",
+    "    data_g = ((mx - i) * m + rs) * i / mx\n",
+    "\n",
```

Comment:
Have you compared with the building `SARIce` composite ?

Reply:
Nope, I looked up the docs but it's not very obvious how to generate a composite and retrieve an image for post processing. How would you do it?

Reply:
To generate the composite, just load it: `scn.load['sarice']`
To do your own post-processing, just use `get_enhanced_image` : http://satpy.readthedocs.io/en/latest/satpy.writers.html?#satpy.writers.get_enhanced_image

Reply:
the enhancements you can put in the `etc/enhancements/generic.yaml` file

Reply:
Strange, I am not able to save the composite dataset. When `scn.save_dataset('sar-ice', 'a1.png') I get a RunTimeError `RuntimeError: No enhancement configuration files found or specified, cannot automatically enhance dataset`. Looks like I can't use `writer.save_dataset` without specifying the enhancer. It doesn't use the existing enhancer `sar-ice` for some reason. I guess I stick to what it is now, and leave the enhancements for the other example.

Reply:
sounds good